src/helpers/getDeckAdvice/advice/KINDREDS_GRACE/spec.js

```diff
@@ -14,17 +26,22 @@ describe('The `KINDREDS_GRACE` advice', () => {
     expect(advice(cards)).to.not.equal(null)
   })
 
-  it('should not be returned if the deck contains Azure Hatchers', () => {
-    const cards = getCards('1n11n21n31n41n51n621n631n81n111n131f101n40')
-    expect(advice(cards)).to.equal(null)
-  })
+  SPAWNS_ARRAY.forEach(spawnCard => {
+    const cardId = spawnCard[0]
```

Comment:
```suggestion
  SPAWNS_ARRAY.forEach(cardId => {
```
This is no longer necessary if using `Object.keys()`.